# Study 849 — Dry January / Veganuary 🍸🥦

**Do the January "Dry January" (abstain from alcohol) and "Veganuary" (go plant-based) waves
move the stocks?**

If the two cultural campaigns shift consumer demand hard and predictably enough, they should
leave a January seasonal in the **abnormal** return (`group − SPY`): the alcohol names
(`BUD STZ TAP DEO SAM`) *down*, plant-based `BYND` *up*, staples `XLP` flat. We read the
January (and February) abnormal return across every year on yfinance daily total-return
closes (1999-01-04 → 2026-06-30, 8 tickers).

*Numbers below are the frozen headline (`docs/results.md`); the live cells run the fast
synthetic control. As-of 2026-06-30, fingerprint `e5210651db30`.*


## 1. The idea in one picture

Millions pledge to skip alcohol (Dry January) or eat plant-based (Veganuary) every January, and drinks makers routinely flag a soft January. If that demand shift reached prices it would show up as a January dip in the alcohol names and a January pop in a plant-based name — measured *against the market* (`− SPY`), so the market's own turn-of-year seasonality is stripped out. Because January is always January, the test is zero look-ahead by construction.

In [1]:
R = dict(alc_jan_mean=-1.2, alc_jan_t1s=-1.28, plant_jan_mean=12.71, plant_jan_t1s=1.23,
         spread_jan_mean=14.46, spread_jan_t1s=1.33, stap_jan_mean=-0.62)
print('January abnormal return (group - SPY):')
print('  alcohol basket : %+.2f%%  (t = %+.2f)  <- Dry January drag' % (R['alc_jan_mean'], R['alc_jan_t1s']))
print('  plant  (BYND)  : %+.2f%%  (t = %+.2f)  <- Veganuary lift (only 7 yrs!)' % (R['plant_jan_mean'], R['plant_jan_t1s']))
print('  staples (XLP)  : %+.2f%%            <- flat control' % R['stap_jan_mean'])
print('  plant - alcohol: %+.2f%%  (t = %+.2f)  <- pure thesis' % (R['spread_jan_mean'], R['spread_jan_t1s']))

January abnormal return (group - SPY):
  alcohol basket : -1.20%  (t = -1.28)  <- Dry January drag
  plant  (BYND)  : +12.71%  (t = +1.23)  <- Veganuary lift (only 7 yrs!)
  staples (XLP)  : -0.62%            <- flat control
  plant - alcohol: +14.46%  (t = +1.33)  <- pure thesis


## 2. Is the detector honest? A live synthetic control

We plant a tunable January abnormal-return seasonal in a seeded toy world (`edge>0`) and check the monthly-seasonality detector recovers it — and that it stays *silent* on the null (`edge=0`, no seasonal). No network.

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from dry_january import data, strategy as st
null = st.synthetic_detect(data.synthetic_panel(edge=0.0, seed=849), month=1)
planted = st.synthetic_detect(data.synthetic_panel(edge=0.05, seed=849), month=1)
print('null world   : Jan dummy NW t = %+.2f  (should be ~0)' % null['t_nw'])
print('planted world: Jan dummy NW t = %+.2f  (should light up)' % planted['t_nw'])

null world   : Jan dummy NW t = +0.75  (should be ~0)
planted world: Jan dummy NW t = +4.09  (should light up)


## 3. The honest verdict — the right footprint, too faint to trade

The folklore points **exactly the right way**: in January the alcohol basket is the year's *most-negative* month (rank 1/12) and `BYND` the *most-positive* (rank 1/12). But the only well-sampled leg — the alcohol drag over 27 Januaries — is just **-1.20%** at *t* = **-1.28**, and the eye-catching **+14.46%** plant-minus-alcohol spread rests on only **7** noisy `BYND` years (*t* = +1.33). The February 'hangover' is absent (+0.12%, *t* = +0.13). Nothing clears |*t*| ≥ 2. The synthetic control shows a *real* +5% seasonal would fire 90% of the time, so this is a genuine but sub-threshold hint. **Signal: Weak. Tradability: Mirage.**